backend/app/ops/user.py

```diff
@@ -33,3 +47,21 @@ def create_user(db: Session, user: schemas.UserCreate) -> models.User:
 
 def check_user_password(user: models.User, password: str) -> bool:
     return bcrypt.checkpw(password.encode("utf-8"), user.hashed_password)
+
+
+def deactivate_user(db: Session, user: User):
+    db.query(models.Sample).filter_by(owner=user.id).delete()
```

Comment:
Deactivation is that user cannot (temporarily) login. Do not delete any of its data